### ISSUE TITLE
Investigate Android 2.x compatibility

### ISSUE THREAD

**Issue by florensie** [w-shackleton/android-netspoof#120]
I downloaded this app and it told me there was an update so I went ahead and downloaded the apk. I deleted the old app before installing and opened the apk but it gave me a parsing error. These are my system specs:
- Model number: GT-S5830i
- Android version: 2.3.6
- Kernel version: 2.6.35.7 dpi@DELL179 #1

I tried to install 2.3.0 and that opened but with that I had problems with using the actual application. I will open another issue for that later today if I can.

**Comment by w-shackleton** [comment]
Thanks. I've recompiled and uploaded [v2.4.0](https://github.com/w-shackleton/android-netspoof/releases/tag/2.4.0) - I've installed this on a couple of devices and it installs successfully. Does this one also work for you?

**Comment by florensie** [comment]
It still gives a parse error. What version of android is it developed for? If the version of the package would mean it is for that version android that would make sense.

**Comment by florensie** [comment]
Maybe it is just a bug with my phone downloading. I'll try downloading from my PC and put it on my sd from there. Also, is it normal that the link for the github download says androidnetspoof-2.4.0-1.apk but the actual downloaded file is called androidnetspoof-2.apk?

**Comment by w-shackleton** [comment]
That might just be the software downloading the file - it might be cutting all other dots from the filename for "security" reasons.

Ah, I recently bumped the minimum version up to 4.0.3 recently - if I recall correctly NS now uses some Android 4.x specific features. I'll check this - I might be able to bump it back down to 2.x in the next version. With the [decline of 2.x devices](https://developer.android.com/about/dashboards/index.html) I decided to deprecate it but I guess there are more root users using 2.x than average.
